modules/mobile/app/controllers/mobile/v0/facility_eligibility_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class FacilityEligibilityController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 3
+      SERVICE_TYPES = %w[primaryCare nutrition podiatry optometry audiology].freeze
+
+      def index
+        require_params
+        unless SERVICE_TYPES.include?(params[:serviceType])
+          raise Common::Exceptions::InvalidFieldValue.new('serviceType', params[:serviceType])
+        end
+
+        page_facilities, page_meta_data = paginate(params[:facilityIds])
+
+        results = Parallel.map(page_facilities, in_processes: page_facilities.size) do |facility|
```

Comment:
I think this would be slightly clearer with facility renamed to facility_id.

Reply:
Yes, makes sense.